common/shell/commands/info_shell.c

```diff
@@ -18,9 +18,9 @@
 #include "plat_version.h"
 
 #ifdef CONFIG_BOARD_AST1030_EVB
-#define BIC_BOARD_NAME "AST1030"
-#elif
-#define BIC_BOARD_NAME "Unknown"
+#define BIC_CHIP_NAME "AST1030"
+#else
+#define BIC_CHIP_NAME "Unknown"
 #endif
```

Comment:
We can use `CONFIG_BOARD` which is defined by Zephyr to get the board we're building for.
This means we don't have to define extra fields.

```
#ifndef CONFIG_BOARD
#define CONFIG_BOARD "unknown"
#endif
.
.
.
shell_print(shell, "* CHIP/OS:       %s - %s", CONFIG_BOARD, RTOS_TYPE);
```